src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -44,25 +39,13 @@ export const DesktopArticleVideoPlayerContent: React.FC<DesktopArticleVideoPlaye
 		<>
 			<div className={styles.desktopArticleVideoTopPlaceholder} ref={placeholderRef}>
 				{adComplete && (
-					<div
-						className={
-							isScrollPlayer ? styles.desktopArticleVideoWrapperScrollPlayer : styles.desktopArticleVideoWrapper
-						}
-					>
-						<div className={styles.topBar}>
-							{!isScrollPlayer && <UnmuteButton />}
```

Comment:
Shouldn't we keep this line as it's rendered for not scrolled player?

Reply:
I'm not sure TBH, I thought it's no longer needed and will be handled by default settings but I might be wrong.

Reply:
Ok, button is brought back, I went to happy with removing stuff :)